multiproof.go

```diff
@@ -274,3 +278,55 @@ func (mp MultiProof) Equal(other MultiProof) bool {
 	}
 	return mp.D.Equal(&other.D)
 }
+
+func groupPolynomialsByEvaluationPoint(fs [][]fr.Element, powersOfR []fr.Element, zs []uint8) [common.VectorLength][]fr.Element {
```

Comment:
Here we receive a very long list of `fs` (i.e: polynomials in evaluation form) that will be evaluated in corresponding `zs`.